src/storage/v2/storage.hpp

```diff
@@ -208,6 +208,7 @@ class Storage {
 
    protected:
     Storage *storage_;
+    std::shared_ptr<Storage> storage_sp_;  // TODO Use a different way of reference counting and unify with raw pointer
```

Comment:
Where do we use it? I can't find initialization. Can we use atomic integer to make reference counting?

Reply:
Not used.